great-idea/css/index.css

```diff
@@ -49,18 +53,224 @@ table {
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
+	text-decoration: none;
 }
 
 html, body {
     height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+	font-family: 'Titillium Web', sans-serif;
+	font-size: 18px;
 }
 
-h1, h2, h3, h4, h5 {
+h1, h2, h3, h4 {
     font-family: 'Bangers', cursive;
-    letter-spacing: 1px;
+    letter-spacing: 3px;
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+
+/* Code was getting messy so wanted to sort it by: Index Only, Services Only, and Both */
+
+/* Index Only */
+.logo {
+	width: 140px;
+	height: 35px;
+	margin-top: 20px;
+	margin-right: 10px;
+}
+
+h6 {
+	font-family: 'Bangers', cursive;
+	margin-bottom: 20px;
+	font-size: 20px;
+}
+
+.headerflexcontainer{
+	margin-top: 50px;
+	display: flex;
+	justify-content: center;
+}
+
+#IOD {
+	margin-top: 120px;
+	margin-bottom: 0px;
+	width: 40%;
+	font-size: 110px;
+	text-align: center;
+}
+
+.headerpic {
+	width: 40%;
+}
+
+.indexflexcontainer{
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	margin: 0px;
+}
+
+.flexitem1 {
+	width: 40%;
+	height: 100px;
+	font-size: 18px;
+	margin-bottom: 15px;
+}
+
+.flexitem2 {
+	width: 26.7%;
+	height: 100px;
+	font-size: 18px;
+	margin-top: 15px;
+}
+
+.middle-img {
+	margin-left: 10%;
+	margin-right: 10%;
+	width: 80%;
+}
+
+.contact {
+	margin-left: 10%;
+	margin-top: 10px;
+	margin-bottom: 55px;
+}
+
+/* Services Only */
+
+.logo2{
+	margin-top: -12.5px;
+	width: 140px;
+	height: 35px;
+	margin-right: 50px;
+}
+
+h5 {
+	font-family: 'Bangers', cursive;
+	margin-top: 15px;
+	font-size: 20px;
+}
+
+.servicesheader {
+	margin-top: 30px;
+	margin-left: 150px;
+	width: 80%;
+	height: 4%;
+}
+
+.section-services {
+	margin-top: 150px;
+}
+
+.services-header-img {
+	margin-left: 10%;
+	margin-right: 10%;
+	width: 80%;
+}
+
+.servicesflexcontainer{
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	margin: 0px;
+}
+
+.flexitem3 {
+	border: solid black 2.5px;
+	width: 38%;
+	height: 265px;
+	background: #DCDCDC;
+	font-size: 18px;
+	padding: 25px;
+	margin: 25px;
+}
+
+.servicesflexcontainer2{
+	display: flex;
+	justify-content: right;
+	justify-content: space-between;
+}
+
+.flexitem4{
+	width: 38%;
+	height: 265px;
+	margin-left: 10%;
+}
+
+.services-info-img{
+	justify-content: left;
+	margin-right: 10%;
+}
+
+/* Both */
+
+nav {
+	display: flex;
+	justify-content: space-evenly;
+}
+
+a {
+	height: 40px;
+	margin-top: 35px;
+	margin-left: 30px;
+	font-size: 15px;
+	color:gray;
+}
+
+nav a:hover {
+	text-decoration: underline;
+}
+
+button {
+	padding: 8px 44px;
+	font-size: 18px;
+	font-family: 'Titillium Web', sans-serif;
+	border: solid black 2.5px;
+	background: white;
+}
+
+#buttonflex{
```

Comment:
A margin-top 50px would have put the button in it's correct spot here. It is currently on top of the text.